mne/evoked.py

```diff
@@ -402,10 +402,11 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             channel traces will not be shown.
         window_title : str | None
             The title to put at the top of the figure window.
-        spatial_colors : bool
-            Color code lines by mapping physical sensor coordinates into color
-            values. Spatially similar channels will have similar colors.
-            Bad channels will be dotted.
+        spatial_colors : None | 'head' | 'skirt' | 'notopo'
```

Comment:
Uh ... no `bool` where False equals None and True defaults to 'skirt'?

Sorry to keep harping on about this ... I just think, semantically, a True is always simpler.

Also, semantically, "skirt" isn't really an argument of "spatial_colors", but of legend, so it's already odd. Allowing a default True would make that a bit easier.